test/support/models.rb

```diff
@@ -215,7 +215,7 @@ class ValidatingUser < User
     only_integer: true
   validates_length_of :name, maximum: 25
   validates_length_of :description, maximum: 50
-  validates_length_of :action, maximum: 10, tokenizer: lambda { |str| str.scan(/\w+/) }
+  validates_length_of :words_in_action, maximum: 10
```

Comment:
Hmmm, I think this may invalidate the test that is checking this. I think we can just remove it. Could you take a look?